python/test/Ice/ami/AllTests.py

```diff
@@ -63,6 +64,7 @@ def ids(self, ids):
 
     def connection(self, conn):
         test(conn is not None)
+        test(type(conn) is IcePy.Connection)
```

Comment:
The real question is: does the Ice for Python API provide symbols in 2 documented modules, Ice and IcePy?

For most language mappings, there is a single namespace, Ice.